libc/lib/CMakeLists.txt

```diff
@@ -35,19 +35,11 @@ foreach(archive IN ZIP_LISTS
   )
   if(LLVM_LIBC_FULL_BUILD)
     target_link_libraries(${archive_1} PUBLIC libc-headers)
+    add_dependencies(${archive_1} libc-startup)
```

Comment:
are there any fullbuild targets we support that don't have startup code yet? (E.g. risc-v linux?)

Reply:
I think `linux` and `gpu` are the only ones with startup code currently. But good catch, should be `if TARGET`.